client/lib/query-args/test/index.js

```diff
@@ -0,0 +1,74 @@
+import page from 'page';
+import useQueryArgs from '../index';
+
+jest.mock( 'react' );
+jest.mock( 'page' );
+
+describe( '#useQueryArgs', () => {
+	const queryArgs = useQueryArgs();
+
+	global.window = {
+		location: {
+			href: '',
+		},
+	};
```

Comment:
Could we switch this file to typescript?

Only major blocker was:

```suggestion
	global.window = { location: { href: '' } } as typeof global.window;
```